docs/development/developer_guide.md

```diff
@@ -47,11 +47,16 @@ a K8s cluster. Set your environment:
 
 ```sh
 export KUBECONFIG=$(echo ~/.kube/config)
```

Comment:
Actually, the `kind create cluster` should do the same setting. So, removing this line would be better.